src/openrct2/ride/Vehicle.h

```diff
@@ -20,12 +20,12 @@
 #include "../common.h"
 #include "../world/map.h"
 
-#pragma pack(push, 1)
 typedef struct rct_vehicle_colour {
     uint8 body_colour;
     uint8 trim_colour;
 } rct_vehicle_colour;
 
+#pragma pack(push, 1)
```

Comment:
just remove it completely, `rct_ride_entry_vehicle` doesn't have its size checked either. (please verify it still loads and ride ratings test works)

Reply:
If I get rid of it, it crashes.